README.md

```diff
@@ -334,10 +334,6 @@ The services included with this setup rely on environment variables to work prop
 | Variable  |  Type | Description | Default Value |
 |---|---|---|---|
 | COURSE_ID | `string` | Demo course id, equivalent to course name or course label. | `intro101` |
```

Comment:
What happens under scenarios where the first use authenticator is still being used? In those cases, we still need these environment variables.

Reply:
I only though that all these variables could be omit since them are calculated or combined with other values, like the 'demo_grader_name' its value is `service-{course_id}`. But I agree, maybe this work can be placed in other changeset.